lib/lightning_web/live/job_live/inspector_form_component.ex

```diff
@@ -74,6 +74,12 @@ defmodule LightningWeb.JobLive.InspectorFormComponent do
     end
   end
 
+  def editor_height(form, id) do
+    height = if form.data.id && id, do: 33.7, else: 38
+
+    "rounded-md border border-secondary-300 shadow-sm overscroll-y-auto h-[calc(100vh_-_#{height}rem)] bg-vs-dark"
+  end
+
```

Comment:
Can you explain the logic here? What are the circumstances the height is different?